test/Utils.test.ts

```diff
@@ -49,6 +49,50 @@ suite('EditorConfig extension', () => {
 					tabSize: 5
 				}
 			},
+			{
+				config: {
+					indent_size: 5
+				},
+				defaults: {
+				},
+				expected: {
+					insertSpaces: 'auto',
```

Comment:
Are you saying you want it to detect indentation when someone has their `indent_size` set to a numeric value? That doesn't make sense to me, because they're basically saying they want spaces by not indicating `indent_size = tab`.